lib/WP_Auth0_Lock10_Options.php

```diff
@@ -132,6 +132,15 @@ protected function build_settings( $settings ) {
     if (isset($settings['language']) && !empty($settings['language'])) {
       $options_obj['language'] = $settings['language'];
     }
+
```

Comment:
Whitespace is off by a space it looks like

Reply:
I'm not sure to understand.